From: Christoph Egger <Christoph.Egger@gmx.de>
Subject: [PATCH] debian/link-against-needed-libs

There are quite some libraries irrlicht needs but does not link
against. This patch makes irrlicht link correctly against all the
needed libraries.

Signed-off-by: Christoph Egger <Christoph.Egger@gmx.de>

---
 lib/irrlicht/source/Irrlicht/Makefile |    2 +-
 1 files changed, 1 insertions(+), 1 deletions(-)

--- a/lib/irrlicht/source/Irrlicht/Makefile
+++ b/lib/irrlicht/source/Irrlicht/Makefile
@@ -88,7 +88,7 @@
 LIB_PATH = ../../lib/$(SYSTEM)
 INSTALL_DIR = /usr/local/lib
 sharedlib install: SHARED_LIB = libIrrlicht.so
-sharedlib: LDFLAGS += -L/usr/X11R6/lib$(LIBSELECT) -lGL -lXxf86vm
+staticlib sharedlib: LDFLAGS += -L/usr/X11R6/lib$(LIBSELECT) -lGL -lXxf86vm -lX11 -lz -lpng -ljpeg -lbz2
 staticlib sharedlib: CXXINCS += -I/usr/X11R6/include
 
 #OSX specific options
